tasks/index.ts

```diff
@@ -6,8 +6,18 @@ const jobsController = JobsController();
 async function run(job: string = process.argv[2], chain: string = process.argv[3]) {
     console.log('Running job', job, chain);
 
-    if (job === 'sync-changed-pools-v3') {
-        return jobsController.addMissingPoolsFromSubgraph(chain);
+    if (job === 'add-pools-v3') {
```

Comment:
Is this class just used so one can run the jobs locally using yarn task ? Maybe add a comment if so